docs/content/connectors/formats/parquet.md

```diff
@@ -0,0 +1,51 @@
+# Protobuf Format
+
+The [Parquet](https://parquet.apache.org/) format allows you to read and write
+Parquet data.
+
+## How to create a table with Parquet format
+
+Here is an example to create a table using the FileSystem connector and Parquet
+format.
+
+If you are using Parquet format in Flink Processor, make sure you have
+configured the environment variable HADOOP_CLASSPATH on both your Flink cluster
+and PyFlink client. Please refer to [Flink's document for Hive
+connector](https://nightlies.apache.org/flink/flink-docs-release-1.16/docs/connectors/table/hive/overview/#supported-hive-versions)
+for supported Hadoop & Hive versions.
+
+Then you can use the Parquet format in filesystem connector as follows.
+
+```python
+schema = (
+    Schema.new_builder()
+    .column("id", types.Int64)
+    .column("val", types.Int64)
+    .column("ts", types.String)
+    .build()
+)
+
+source = FileSystemSource(
+    name="filesystem_source",
+    path="...",
+    data_format="parquet",
+    schema=schema,
+)
+```
+
+## Data Type Mapping
+
+The following table lists the type mapping from Feathub type to Parquet type.
+
+| Feathub type | Parquet type | Parquet logical type |
+| ------------ | ------------ | -------------------- |
+| Bytes        | Binary       | UTF8                 |
+| String       | Binary       | UTF8                 |
+| Int32        | Int32        |                      |
+| Int64        | Int64        |                      |
+| Float32      | Float        |                      |
+| Float64      | Double       |                      |
+| Bool         | Boolean      |                      |
+| VectorType   | -            | LIST                 |
```

Comment:
What does "-" mean here?

Reply:
Let's keep the discussion in the previous comment.